geode-docs/tools_modules/gfsh/command-pages/search.html.md.erb

```diff
@@ -0,0 +1,49 @@
+---
+title:  search lucene
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## <a id="search_lucene" class="no-quick-link"></a>search lucene
+
+Search a lucene index
```

Comment:
Is there more to this command?  I don't know what it does from this description. There's no output shown for the sample command, so that doesn't help me.   There are probably command line options that specify what to search for and which index to search within.

Capitalize 'Lucene?'